Maple2.Server.Game/Packets/UserEnvPacket.cs

```diff
@@ -40,4 +41,16 @@ private enum Command : byte {
 
         return pWriter;
     }
+
+    public static ByteWriter LoadClaimedRewards(IDictionary<int, bool> claimedRewards) {
+        var pWriter = Packet.Of(SendOp.UserEnv);
+        pWriter.Write<Command>(Command.MasteryRewardsClaimed);
+        pWriter.WriteInt(claimedRewards.Count);
+        foreach (KeyValuePair<int, bool> reward in claimedRewards) {
```

Comment:
I generally prefer decomposition since you can name the different values: `KeyValuePair<int, bool>` => `(int rewardId, bool isClaimed)`